definitions/npm/ramda_v0.x.x/flow_v0.39.x-/ramda_v0.x.x.js

```diff
@@ -9,6 +9,7 @@ type Transformer<A,B> = {
 
 declare module ramda {
   declare type UnaryFn<A,R> = (a: A) => R;
+  declare type UnaryPromiseFn<A,R> = (a: A) => Promise<R>;
```

Comment:
We could reuse `UnaryFn` here:

```js
declare type UnaryPromiseFn<A,R> = UnaryFn<A, Promise<R>>
```

It seems like we could do that to match the style of `UnarySameTypeFn` and `BinarySameTypeFn` below, which reuse `UnaryFn` and `BinaryFn`.